src/scenes/Wallet/components/ChangPassphraseDialog/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Dialog, { DialogTitle, DialogContent, DialogActions } from 'material-ui/Dialog';
+import { TextField, Button, Typography } from 'material-ui';
+import { withStyles } from 'material-ui/styles';
+import { FormattedMessage, injectIntl, intlShape, defineMessages } from 'react-intl';
+import _ from 'lodash';
+
+import appActions from '../../../../redux/App/actions';
+
+const messages = defineMessages({
+  oldPassphrase: {
+    id: 'str.oldPassphrase',
+    defaultMessage: 'old passphrase',
+  },
+  newPassphrase: {
+    id: 'str.newPassphrase',
+    defaultMessage: 'new passphrase',
+  },
+});
+
+
+@injectIntl
+@withStyles(null, { withTheme: true })
+@connect((state) => ({
+  walletAddresses: state.App.get('walletAddresses'),
+}), (dispatch) => ({
+  changePassphrase: (oldPassphrase, newPassphrase) => (dispatch(appActions.changePassphrase(oldPassphrase, newPassphrase))),
+}))
+export default class ChangePassphraseDialog extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired, // eslint-disable-line react/no-typos
+    dialogVisible: PropTypes.bool.isRequired,
+    onClose: PropTypes.func.isRequired,
+    closeEncryptDialog: PropTypes.func.isRequired,
+    changePassphrase: PropTypes.func.isRequired,
+  };
+
+  state = {
+    oldPassphrase: '',
+    newPassphrase: '',
+  };
+
+  submitPasswordChange = () => {
+    const { oldPassphrase, newPassphrase } = this.state;
+    this.props.changePassphrase(oldPassphrase, newPassphrase);
+    this.props.closeEncryptDialog();
+    this.props.onClose();
```

Comment:
destructure these: `const { changePassphrase, closeEncryptDialog, onClose } = this.props;`